contracts/libraries/MathUtilsV2.sol

```diff
@@ -0,0 +1,47 @@
+pragma solidity ^0.5.11;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+library MathUtils {
```

Comment:
Curious as to why the decision was made to create a V2 file rather than adjusting the existing MathUtils contract. 

No state is affected by this change as these are all pure functions. 

Purely historical reference of code changes to the protocol?

Reply:
LIP-40 only updates the Minter contract to use a new MathUtils library and to accomplish this the Minter is updated to import `MathUtilsV2.sol`. Other contracts such as the BondingManager will continue to use the old MathUtils library by continuing to import `MathUtils.sol`.

Reply:
I see. I feel like the BondingManager could also benefit from more precision but that's out of scope for this PR then.